### PR TITLE
Use polymer-cli for linting/testing

### PR DESCRIPTION
**Pull request by ryantmer**
Allows us to use the polymer-2-hybrid ruleset.

### CODE REVIEW

**Comment by dlockhart** [comment]
Might be worth bumping up to El Captain (OSX 10.12) in `wct.conf.js`?

File: package.json [hunk 1]
```diff
@@ -6,11 +6,10 @@
     "build": "browserify -g uglifyify d2l-ajax-framed.js > d2l-ajax-framed.min.js",
     "postinstall": "bower install",
     "serve": "polyserve --cors",
-    "test:lint:js": "eslint test/ --ext .js",
-    "test:lint:html": "eslint *.html",
-    "test:lint": "npm run test:lint:js && npm run test:lint:html",
-    "test:no-lint": "wct --skip-plugin sauce -p",
-    "test": "npm run test:lint && npm run test:no-lint"
+    "test": "npm run test:lint:js && npm run test:lint:wc && npm run test:unit",
+    "test:lint:js": "eslint --ext .js,.html .",
```
Comment (dlockhart) [inline package.json]: The only thing we may have lost here is eslinting of the JavaScript inside the WC HTML files... does polymer test do that for us?

Comment (dlockhart) [inline package.json]: Can you confirm the eslinting is still catching stuff inside the `/test` directory?

Comment (ryantmer) [inline package.json]: Oh boyo does it ever not! Nice catch. Also realized we're not using the Brightspace config here, so a few more small changes coming up.

**Comment by ryantmer** [comment]
The `?w=1` is probably going to be helpful on the diff... didn't realize we haven't been using `eslint-config-brightspace` here, but fortunately it's mostly just spaces and missing semicolons.

**Pull request merged**